tests/unittest_builder.py

```diff
@@ -191,6 +191,114 @@ class C:
             assert c.fromlineno == 13
         assert c.tolineno == 14
 
+    @staticmethod
+    def test_class_with_docstring() -> None:
+        """Test class nodes which only have docstrings."""
+        code = textwrap.dedent(
+            '''
```

Comment:
```suggestion
            '''\
```
I've started adding backslashes to the first line. That way it starts counting with `1` like you would expect, instead of `2`. It's not something I would update all tests for, but useful for new ones.

Reply:
Very clever !